web/WEB-INF/web.xml

```diff
@@ -14,9 +14,41 @@
     <description>An address where OpenGrok can receive new configuration</description>
   </context-param>
 
+  <!--
```

Comment:
I actually wonder if it is better to have this commented or uncommented.
Lots of folks have scripts that just come to web.xml and seds the values, which is easy.
However uncommenting and XML section is harder ... so I guess ... leave it uncommented for people to be able to hack this easily ...